tests/playbooks/roles/install-csi-cinder/tasks/main.yaml

```diff
@@ -153,6 +153,17 @@
       kubectl apply -f manifests/cinder-csi-plugin
   ignore_errors: yes
 
+- name: Deploy snapshot manifests
+  shell:
+    executable: /bin/bash
+    cmd: |
+      kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/release-4.2/client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml
```

Comment:
@jichenjc how about just adding one line for more readability

` kubectl create -f https://github.com/kubernetes-csi/external-snapshotter/blob/release-4.2/client/config/crd/` or

we can use the same as mentioned in readme here, https://github.com/kubernetes-csi/external-snapshotter#usage

Reply:
@ramineni you are suggesting add one line for comment or 
use following because it's a folder ? 
https://github.com/kubernetes-csi/external-snapshotter/blob/release-4.2/client/config/crd/

Thanks :)

Reply:
@ramineni 

error in apply the dir in remote ...

```
# kubectl create -f https://github.com/kubernetes-csi/external-snapshotter/blob/release-4.2/client/config/crd/
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```

Reply:
I guess what @ramineni means is to use kustomize for github folders, something like:

```
kubectl apply -k 'github.com/kubernetes-csi/external-snapshotter/client/config/crd?ref=release-4.2'
```

Reply:
ok, updated.. thanks